### ISSUE TITLE
Confusing error message when you update a m1 Device an Enter no Credentials.

### ISSUE THREAD

**Issue by colorenz** [grahampugh/erase-install#149]
Hey

If you do not enter a password when updating an M1 Device, you will receive a confusing error message for the. Since the text says reinstall and not update. This could confuse the user.

 /Library/Management/erase-install/erase-install.sh --os=12 --build=21A559 --reinstall --no-fs --check-power --power-wait-limit 180 --min-drive-space 45 --current-user --depnotify --update --pkg...
 
The following text is currently used.

```
# Dialogue localizations - invalid user
dialog_user_invalid_en="This account cannot be used to to perform the reinstall"
dialog_user_invalid_de="Dieses Konto kann nicht zur Durchführung der Neuinstallation verwendet werden"
dialog_user_invalid_nl="Dit account kan niet worden gebruikt om de herinstallatie uit te voeren"
dialog_user_invalid_fr="Ce compte ne peut pas être utilisé pour effectuer la réinstallation"

```

Maybe we can adjust the text a bit or add a other text for the update usecase.

```
# Dialogue localizations - invalid user
dialog_user_invalid_en="This account cannot be used to to perform the **update**/reinstall"**
dialog_user_invalid_de="Dieses Konto kann nicht zur Durchführung **des Updates oder Neuinstallation** verwendet werden"
dialog_user_invalid_nl="Dit account kan niet worden gebruikt om de herinstallatie uit te voeren"
dialog_user_invalid_fr="Ce compte ne peut pas être utilisé pour effectuer la réinstallation"

```

 
```

 [check_newer_available] No newer builds found
yes username is a member of staff
   [get_user_details] username is a Volume Owner
<dscl_cmd> DS Error: -14090 (eDSAuthFailed)
   [check_password] ERROR: The password entered is NOT the login password for username.
button returned:OK
```

![image](https://user-images.githubusercontent.com/16665557/140744855-0443ccbf-4c74-4d5e-aaf7-b8ee3cfaea72.png)

**Comment by grahampugh** [comment]
This came up elsewhere.  In Recovery OS, it has always been described as "Reinstall macOS" - this is why I used the work "reinstall". But a suggestion was to use the word "install" / "installation", which is what Apple have in their own dialogs for software update.

I don't intend to have different options for "upgrade" and "reinstall" because it is the same process (reinstalling the OS), and people are going to get confused which option to use.

**Comment by colorenz** [comment]
Yeah install would be fine.

**Comment by grahampugh** [comment]
Should be fixed now in v25.0. Let me know if not.
